src/main/java/ee/ituk/api/common/exception/NotFoundException.java

```diff
@@ -7,12 +7,12 @@
 public class NotFoundException extends ApiException {
 
     public NotFoundException() {
-        this.status = HttpStatus.PRECONDITION_FAILED;
+        this.status = HttpStatus.NOT_FOUND;
         this.error = "Object not found";
     }
 
     public NotFoundException(List<ErrorMessage> errorMessages) {
-        this.status = HttpStatus.PRECONDITION_FAILED;
+        this.status = HttpStatus.NOT_FOUND;
```

Comment:
Siis see annab 404, see pigem tähendab et lehe pole

Reply:
404 - resource does not exist, see ei ole täpne siin?